lib/resque/pool/version.rb

```diff
@@ -1,5 +1,5 @@
 module Resque
   class Pool
-    VERSION = "0.4.0.dev"
+    VERSION = "1.0.0.skroutz.0"
```

Comment:
Why do we bump to 1.0.0? We're actually still based on upstream 0.4.0-pre AFAICT.

Reply:
@apoikos I am not sure what the correct approach is here. 
According to semver https://semver.org/ when introducing changes in a backwards-compatible manner then we should bump the `minor` version. But we also introduced a breaking change. This is the change of the YAML config file. This file is like our `Public API` and when breaking changes occur to the public api the `major` version should be bumped.
As i said i am not sure what the correct approach is here, if you think the new version should be `0.4.0.skroutz.0` then i am totally fine with that.

Reply:
Even if we break the existing API it's a good practice to have a version that shows the upstream version we forked.

I vote for `0.4.0.skroutz.0`.

Reply:
Done.